app/full_text_search.py

```diff
@@ -101,6 +101,31 @@ def create_query_txt(query_txt):
     return enclose_in_parenthesis(romanized_query)
 
 
+def get_person_ids_from_results(romanized_query, results_list):
+    """
+    Returns person record_id (a part of romanized_query matches person name) list.
```

Comment:
"Returns record_id of persons whose name contains at least one word in romanized_query."

Also, please mention that:

- Why you need to apply this regexp
- It also removes dups (i.e., If results_list contains multiple results with the same index_results, it returns just one of them.)